### PR TITLE
Compiling Headlamp app on Windows fixes

### PR DESCRIPTION
**Pull request by illume**
# There were a few issues when building the app on windows

export, mkdir -p, and `''` quoting were problematic

Additionally. the github app windows build now doesn't use bash with npm. This is a test which shows this PR works without bash on windows now.

## Testing done

- [x] `npm run dev` starts and seems to work
- [x] generated app installs, starts and seems to work 

ps. Note: server.exe will hang around on app close. So make sure to kill that with task manager if you are testing.  There's a PR for that here: https://github.com/kinvolk/headlamp/pull/395

### CODE REVIEW

**Review by joaquimrocha — APPROVED** [review]
While it's good that we can make the app work on Windows without bash, I wonder if it's worth doing it. I am concerned with the fact that future changes to the build scripts will always need to use extra tools to ensure compatibility, instead of just using bash as a wrapper/standard for building it.

Is there a good argument for not using bash?

**Comment by illume** [comment]
Bad DX. Bash is not standard on windows. Even when using bash, npm doesn't use bash by default.

**Comment by ashu8912** [comment]
> Bad DX. Bash is not standard on windows. Even when using bash, npm doesn't use bash by default.

That is true at times I have to change the shell for windows manually to bash for npm scripts to work

File: app/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "homepage": ".",
   "productName": "Headlamp",
   "scripts": {
-    "compile-electron": "babel electron --out-dir electron/ --extensions '.ts'",
+    "compile-electron": "babel electron --out-dir electron/ --extensions .ts",
```
Comment (ashu8912) [inline app/package.json:9]: @illume How is this probelmatic, I tried with both ways without and with quotes and it works

**Review by ashu8912 — COMMENTED** [review]
Does ```cross-env``` needs to be an app dependency?? 
Shouldn't it be a dev-dependency?

**Review by ashu8912 — COMMENTED** [review]
same here mkdirp should be a dev-dependency

File: app/package.json (hunk 1 above)
Comment (illume) [reply app/package.json:9]: It runs, but does not compile the scripts (reports 0 compiled and should be something like 4). Note, this is when using cmd.

**Comment by illume** [comment]
@ashu8912 `cross-env` and `mkdirp` are required to build for production. For that reason I think they are correctly placed as a dependency.

**Review by ashu8912 — APPROVED** [review]

File: app/package.json (hunk 1 above)
Comment (ashu8912) [reply app/package.json:9]: aah I was using the wrong shell, with cmd it is a problem

**Pull request merged**